fastlane/lanes/localization.rb

```diff
@@ -253,14 +254,6 @@
       strings_file_path: 'bundle/android/strings.xml',
       source_id: 'gutenberg'
     },
-    {
-      name: 'Login Library',
-      import_key: 'wordpress-login',
-      repository: 'wordpress-mobile/WordPress-Login-Flow-Android',
-      strings_file_path: 'WordPressLoginFlow/src/main/res/values/strings.xml',
-      exclusions: ['default_web_client_id'],
-      source_id: 'login'
-    },
```

Comment:
👋 @AliSoftware @mokagio WordPress-Login-Flow will be soon part of this repository. Will this change simply work® or do we have to adjust this somewhere else as well?

Reply:
Yes. Given you made this change in conjunction with the other change you made on like 247 so that the `strings.xml` from that newly-local Login library are now pulled from the local module instead of the remote repo, I think this should just work® 🙂

Reply:
Thanks!